# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../attr/install.patch
# Copyright (C) 2009 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

our install wrapper deletes the library and crashes, because bash relies on it

--- ./install-sh.orig	2009-11-17 10:58:36.000000000 +0100
+++ ./install-sh	2009-11-17 10:59:49.000000000 +0100
@@ -92,7 +92,7 @@
 
 if $INSTALL
 then
-    CP=cp; LN=ln; MKDIR=mkdir; CHMOD=chmod; CHOWN=_chown
+    CP=install; LN=ln; MKDIR=mkdir; CHMOD=chmod; CHOWN=_chown
 else
     CP=true; LN=true; MKDIR=true; CHMOD=true; CHOWN=true
 fi
